packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -284,7 +281,7 @@ def get_output_parameters_with_type(
 
         output_parameters_with_type.append(f"{parameter_name}: {parameter_type}")
 
-    return f"{os.linesep}    ".join(output_parameters_with_type)
+    return f";".join(output_parameters_with_type)
```

Comment:
I don't think generating worse output prior to reformatting with Black is an improvement. Revert this.

Reply:
Updated the method to return a list and used for loop in mako to generate the Output class.